src/main/java/frc/robot/utils/PIDSettings.java

```diff
@@ -0,0 +1,106 @@
+package frc.robot.utils;
+
+/**
+ * This class is used to store settings for different PIDs
+ */
+public class PIDSettings {
+
+  private double KS;
+  private double KP;
+  private double KI;
+  private double KD;
+  private double KV;
+  private double KA;
+  private double tolerance;
+  private double deltaTolerance;
+
+  /**
+   * @param KP        The the Proportional coefficient of the PID loop in this
+   *                  command.
+   * @param KI        The Integral coefficient of the PID loop in this command.
+   * @param KD        The Differential coefficient of the PID loop in this
+   *                  command.
+   * @param tolerance The error tolerance of this command.
+   * @param deltaTolerance  The time this PID loop will wait while within tolerance of
+   *                  the setpoint before ending.
```

Comment:
@param deltaTolerance  The time this PID loop will wait while within tolerance of
TO
@param deltaTolerance The time this PID loop will wait while within tolerance of